src/main/java/duke/commands/DoneCommand.java

```diff
@@ -0,0 +1,42 @@
+package duke.commands;
+
+import duke.storage.Storage;
+import duke.tasks.Task;
+import duke.tasks.TaskList;
+import duke.ui.UI;
+
+/**
+ * Represents a Done command for when the user wants to mark
+ * a task as done.
+ */
+public class DoneCommand extends Command {
+    public static final String COMMAND_WORD = "done";
+    public final int sizeOffset = -1;
```

Comment:
Hmm, this seems to be a constant, maybe you'd like to uppercase it with underscore? 💇

Reply:
Don't know how I missed that, Thanks! 😄